### DISCUSSION TITLE
real time collaboration [Sep 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3919] [Discord Threads]
> 🙋‍♂️ **Bergi9**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### this feature is not yet possible, we will be working on this soon.  
#  
> 🙋‍♂️ **Bergi9**  
#### Ok, so the socket.io is just implemented for the realtime collaboration anytime soon?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020943932433121361)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by mantianxinglzm** [comment]
now still cannot reload automatically
